elliottlib/bzutil.py

```diff
@@ -1058,6 +1058,14 @@ def get_highest_security_impact(bugs):
     return 'Low'
 
 
+def sort_cve_bugs(bugs):
+    def cve_sort_key(bug):
+        impact = constants.security_impact_map[get_highest_security_impact([bug])]
+        year, num = bug.alias[0].split("-")[1:]
+        return impact, -int(year), int(num)
+    return sorted(bugs, key=cve_sort_key, reverse=True)
```

Comment:
How are `Low` vs `Critical` vs `Urgent` impacts sorted here?

Reply:
they will be translated to number by https://github.com/Ximinhan/elliott/blob/a898f60bd397ddf8130a6e2e658bb2c0ffd01225/elliottlib/constants.py#L33
so Critical and Urgent have the same value